StRoot/StRHICfUtil/StRHICfFunction.h

```diff
@@ -0,0 +1,72 @@
+/***************************************************************************
+ * Author: Seunghwan Lee
+ ***************************************************************************
+ *
+ * Description: RHICf Useful class for calibration and reconstruction
+ *
+ ***************************************************************************
+ * $Id: StRHICfFunction.h,v 1.0 2022/08/25 2:51:00 SeunghwanLee Exp $
+ * $Log: StRHICfFunction.h,v $
+ ***************************************************************************/
+
+#ifndef StRHICfFunction_H
+#define StRHICfFunction_H
+
+#include <iostream>
+#include <TString.h>
+#include "StRoot/StEvent/StEnumerations.h"
+
+class StEnumerations;
+
+enum RHICfRunType
+{
+	kRHICfTL = 0, 
+	kRHICfTS = 1, 
+	kRHICfTOP = 2 
+};
+
+enum RHICfIsValue
+{
+	kRHICfOk = true,
+	kRHICfFatal = false
+};
+
+enum RHICfParticle
+{
+    kRHICfPhoton = 0,
+    kRHICfHadron = 1
+};
+
+class StRHICfFunction 
+{
+	public: 
+		StRHICfFunction();
+		virtual ~StRHICfFunction();
+
+        void initChecker();
+
+		void setRunType(Int_t type){mRunType = type;}
+		Int_t getRunType(){return mRunType;}
```

Comment:
This appears to be a utility class, rather than a persistent data class.  The preference is to use standard c++ types (int, float, etc...) rather than the ROOT tyles (Int_t, Float_t, etc...).

Also, because this class is a persistent class (i.e. saved to disk) you do not need the ClassDef statement below.

Reply:
This class is a utility class, not a saving class.

**Also, because this class is a persistent class (i.e. saved to disk) you do not need the ClassDef statement below.**

as per your comment, if a class is "a utility class", there is no need for the ClassDef, it's right?

Reply:
Correct.  If you do not write it to disk (e.g. it is not part of the MuDst or StEvent) then you do not need to declare the ClassDef.

Reply:
Thank you for the answer.